dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'pm_utils'
-version: '1.4.1'
+version: '1.4.2'
```

Comment:
Since we remove code it should probably be a 2.0.0 release (breaking changes).

Reply:
Changed